tools/parity_scale_cli/src/main.rs

```diff
@@ -113,26 +135,34 @@ struct ScaleJsonArgs {
     type_name: String,
 }
 
+fn is_colouring_supported() -> bool {
```

Comment:
Nit: let's use "color" instead of "colour", and change it in other places too.

```suggestion
fn is_color_supported() -> bool {
```

Reply:
Looks good otherwise, thank you!

Reply:
thanks, fixed https://github.com/hyperledger/iroha/pull/4680/commits/f8da8a2bde71e491263aff868caaf98e9f1742ec

`cli/src/lib.rs` also has colour typo too. -> [git](https://github.com/hyperledger/iroha/blob/main/cli/src/lib.rs#L778-L783).
Should I address this in this PR? or is creating another one more suitable?

Reply:
I think it's fine to address it here

Reply:
thanks, fixed -> https://github.com/hyperledger/iroha/pull/4680/commits/9c55d823f478ea331a37afaa716c74ddad286171